src/components/SRNEntry.vue

```diff
@@ -247,10 +248,29 @@ export default {
       this.validateCount = userValidationResponse.validateCount;
       this.invalidLoginMessage = userValidationResponse.invalidLoginMessage;
       this.isLoading = false;
+
+      //clear the input field if entry is incorrect
+      if(this.invalidLoginMessage != ""){
+        this.getLatestEntry["userID"] = "";
+      }
+
+      //incorrect entry the first time
+      if(!this.isUserValid && this.validateCount == 1){
+        var purposeParams = "incorrect-entry"
+        var tempUserIDList = [{userID: userID.toString(), valid: this.isUserValid}]
```

Comment:
because the message being sent is specific to each userID. And this is the format the lambda handler parses the input. If I just send the userID, then I'll need to add a whole other logic for the lambda.